app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -110,7 +110,7 @@ class MainActivity : AppCompatActivity() {
 
     private fun checkAndLoadFragment(fragment: Fragment) {
         val savedFragment = supportFragmentManager.findFragmentByTag(fragment::class.java.name)
-        if (savedFragment != null) {
+        if (savedFragment != null && fragment !is SearchFragment) {
```

Comment:
This is not a fix. This is a workaround

Reply:
> This is not a fix. This is a workaround

But as in Eventbright everytime open search section, it opens with a new fragment so, I think no saved state is required for search fragment. That's why I direct load with new fragment if the search fragment is open. Should I fix this issue in another way?

Reply:
Yes, the special case should be handled in a generic way while opening search fragment, not a special case in a method which should not even know about the existence of search fragment